### DISCUSSION TITLE
Bar Chart needed

### DISCUSSION THREAD

**Discussion by eflann** [twilio-labs/paste#1794] [Q&A] [answered]
What is the equivalent of Bar Chart -> https://code.hq.twilio.com/react/react-components/blob/master/packages/ui-component-chart/docs/BarChart.md in paste?
I'm not seeing it when I search here https://paste.twilio.design/?

**Comment by TheSisb** [comment, marked as answer]
Hi @eflann,

Paste doesn't have any charting components or libraries at this time. Our recommendation would be to use the existing charting tools that exist in your product until we have something to replace it, like the one you linked.

**Reply by eflann** [reply]
@TheSisb Do you have a timeline for when charting components will be added to paste?

**Reply by SiTaggart** [reply]
I would say there is a high degree of chance that it might not ever get added, certainly not in the way we maybe used to.

We might just say "use highcharts" and show folks how to configure it and use it. We might provide palettes, guidelines, and examples but not a component so to speak
